### ISSUE TITLE
Unity6/ Android URP/ CRASH

### ISSUE THREAD

**Issue by MikkCzs** [suriyun-production/mmorpg-kit-docs#2680]
Hello, does anyone here work with Unity6 and Android URP?

After installing the MMO Kit, the application crashes immediately upon launch with a red screen (shader missing color).

I’ve tried a fresh project, which worked fine on the phone, but after importing the kit into Unity, the app crashes again immediately upon launch.

This is not an issue with URP settings or materials. I’ve also tried different build settings, but there’s no change. Even an empty scene with just a background stops working this way.

**Comment by insthync** [comment]
How did you make sure it is not because of URP?

Did you try to debug it?, did you try to use logcat to capture error messages?

**Comment by MikkCzs** [comment]
> How did you make sure it is not because of URP?
> 
> Did you try to debug it?, did you try to use logcat to capture error messages?

Hello. Are you the developer of this Kit?  
Your question is unnecessary; of course, I’ve tried multiple things, and if the issue were elsewhere, I wouldn’t be writing here. There is a BUG in the Kit that manifests in Unity 6!

**Comment by insthync** [comment]
Yes I am, and I want to know why it is not necessary, how can I find source of issues without those info?

**Comment by MikkCzs** [comment]
> Yes I am, and I want to know why it is not necessary, how can I find source of issues without those info?

It’s unnecessary to ask if I’ve tried these things. Of course, before posting on the forum, I tried every possible combination of solutions—from a clean project to RP changes, reinstalls, and Build setting adjustments.  

The issue always occurs after installing the plugin. The error is triggered the moment the application starts (before the initial logo).  

The question was whether anyone here is using Unity 6 and working with URP for Android. So, do you have this setup active, and are you able to successfully build? Is that correct?

**Comment by insthync** [comment]
Then there is no error log messages? Or something like that?

**Comment by MikkCzs** [comment]
> Then there is no error log messages? Or something like that?

No, there’s no error message. The application immediately crashes when launched on Android (tested on multiple devices). After the black screen at startup—just as the Unity logo is supposed to appear—it crashes.  

There are no issues in the editor, either during Play mode or when building in the editor.  

The problem occurs with any scene that includes the MMORPG Kit in the project (even empty scenes behave the same way when built).

**Comment by insthync** [comment]
Can you use logcat to capture error log messages

**Comment by MikkCzs** [comment]
> logcat

ok i install it and capture

**Comment by insthync** [comment]
ROG 3

**Comment by insthync** [comment]
ROG 3

**Comment by MikkCzs** [comment]
> ROG 3

Nice..Thanks!

**Comment by MikkCzs** [comment]
Hello,  

Is it possible to find an updated manual for the installation and the kit? The one I have (the doc in the asset) seems outdated.  
When installing the plugin for Unity6 - Android, did you follow the manual and overwrite the Unity settings? I found an issue, but it’s not possible to install the kit according to the instructions, and certain parts of the Unity settings must not be overwritten.

**Comment by insthync** [comment]
https://suriyun-production.github.io/mmorpg-kit-docs/#/pages/181-urp-android-pink
